x-pack/plugin/sql/sql-action/src/test/java/org/elasticsearch/xpack/sql/action/SqlQueryRequestTests.java

```diff
@@ -110,4 +112,11 @@ protected SqlQueryRequest mutateInstance(SqlQueryRequest instance) {
         mutator.accept(newRequest);
         return newRequest;
     }
+
+    @Test
+    protected void testTimeZone() {
```

Comment:
Thanks. Could you make this method public and remove the annotation please? We don't use the "@Test" annotation in our tests, although I know it is common elsewhere, but it will fail our code style check. Also, could you rename the test method to something making clear this tests only the exception, e.g. testTimeZoneNullException.